solutions/swb-app/src/staticRouteConfig.ts

```diff
@@ -120,7 +122,7 @@ export const routesMap: RoutesMap = {
       }
     ]
   },
-  '/environmentTypes/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}': {
+  [`/environmentTypes/${uuidRegExp}`]: {
```

Comment:
`environmentType` is not updated with a prefix here, because `environmentType` id will be updated as part of GALI-1839